### PR TITLE
Clear log in es3fFragmentOutputTests.js

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

**Comment by zhenyao** [comment]
lgtm

File: sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js [hunk 1]
```diff
@@ -1389,7 +1389,7 @@ var tcuImageCompare = framework.common.tcuImageCompare;
             tcuTestCase.runTestCases();
         } catch (err) {
             testFailedOptions('Failed to es3fFragmentOutputTests.run tests', false);
-            console.log(err);
+            bufferedLogToConsole.log(err);
```
Comment (kenrussell) [inline sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js]: Typo: bufferedLogToConsole.log -> bufferedLogToConsole

**Comment by kenrussell** [comment]
Thanks -- one typo.

File: sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js]: Fixed here: https://github.com/KhronosGroup/WebGL/pull/1901

Comment (qkmiao) [inline sdk/tests/deqp/functional/gles3/es3fFragmentOutputTests.js]: Thanks.  Fixed in #1902

**Pull request merged**